chapters/background.tex

```diff
@@ -1,4 +1,11 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
-
+TensorFlow \cite{tensorflow} is a open source machine learning library developed by Google.
+It supports a wide variety of programming languages, mainly targeting web and mobile applications.
+For mobile development TensorFlow provides a Lite version called TensorFlow Lite (TFLite) which can be easily included in a mobile app.
+The main advantage TFLite offers, is the ability to add a self trained model to detect smiles.
+With this being said, the app's TFLite model can be broken down into just being able to detect smiles, rather than having the whole emotion detection.
```

Comment:
I think it should be mentioned that emotions detection is possible with TF before referring to that fact. So I'd still love to have something like this instead:
The advanced emotion detection feature of TF can be reduced to a simpler smile detection model, which matches the needed functionality.